nservicebus/messaging/messages-events-commands.md

```diff
@@ -43,42 +41,43 @@ There are checks in place to ensure best practices are followed. Violations of t
 partial: errors
 
 
- ## Designing messages
+## Designing messages
 
- Messages represent data contracts between endpoints. They should be designed according to the following guidelines.
+Messages represent data contracts between endpoints. They should be designed according to the following guidelines.
  
- Messages should:
+Messages should:
+
+* be simple [POCO](https://en.wikipedia.org/wiki/Plain_old_CLR_object) types.
+* be as small as possible.
+* satisfy the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle). Types used for other purposes (e.g. domain objects, data access objects, or UI binding objects) should not be used as messages.
 
- * be simple [POCO](https://en.wikipedia.org/wiki/Plain_old_CLR_object) types.
- * be as small as possible.
- * satisfy the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle). Types used for other purposes (e.g. domain objects, data access objects, or UI binding objects) should not be used as messages.
+Note: A message must be a reference type (`class`). An attempt to send a message which is a value type (`struct`) will result in a runtime exception.
```

Comment:
Which runtime exception? That would help with SEO.

Reply:
```
System.NullReferenceException : Object reference not set to an instance of an object.
   at NServiceBus.TypeExtensionMethods.SerializationFriendlyName(Type t)
   at NServiceBus.XmlSerialization.Serialize()
   at NServiceBus.XmlMessageSerializer.Serialize(Object message, Stream stream)
   at 
```

oops :)

I'll fix this in the core so until then let's leave this as is?